### DISCUSSION TITLE
More ideas for the daily challenge

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#28694] [Ideas] [answered]
I didn't read most of the info about the DC, but i decided to post this in case this already wasn't there, and I find these ideas a good thing to add

All stuff here is regarding streaks for the DC

1. **Rankings** - add a leaderboard for the DC, specifically streaks. It's not something competitive too much, but imo it would be a fun feature to see how many people play Osu daily and for how long. Could be 'hidden' like the kudosu rankings and not displayed in user profile, but that's for you to decide (personally, I think the better option would be displaying it next to the global and country ranks on user profile)
2. **Max Streak display on user profile** - another thing that's self-explanatory. Could be listed near the stats on user profile. It's probably already going to be added and I'm blind, but I wanted to put it here just in case
3. **Osu! Regular rank** - it would display the similar way to the BN, GMT ect. ranks. You would automatically get this rank from having a big streak (I'd say half a year and up, but a another good option, is a whole year and up, so it would be more exclusive)

That's it for now I think. If I find any more ideas, I'll most likely just edit the post again 
Thanks for reading

**Comment by bdach** [comment, marked as answer]
We are doing (2). (1) and (3) is technically possible if we get enough interest.
